Release v0.9.0 (#103)
meilisearch/documentation: guides/advanced_guides/synonyms.md

# Synonyms

To make your search more relevant you can [create a list of synonyms](/references/synonyms.md#update-synonyms).

> If your database contains a lot of a word that has on or more popular synonyms, by configuring the synonyms the words can be associated. **This allows the same search results if you write one or the other word**.

Reviewer: ```suggestion
If your database contains a lot of a word that has one or more popular synonyms, by configuring the synonyms the words can be associated. **This allows the same search results if you write one or the other word**.
```

> If your database contains a lot of a word that has one or more popular synonyms, by configuring the synonyms the words can be associated. **This allows the same search results if you write one or the other word**.

Reviewer 2: I don't think is more clear. But apply yours and we will see that later

> By searching `phone` you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, the data containing `phone` will not change your results.

Reviewer: ```suggestion
By searching `phone` you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, documents containing `phone` will not be shown in your results.
```
